cli/flox/src/utils/errors.rs

```diff
@@ -22,7 +22,7 @@ pub fn format_error(err: &EnvironmentError2) -> String {
 
         // todo: enrich with a path?
         EnvironmentError2::EnvNotFound => formatdoc! {"
-            Found a '.flox' directory, but were unable to locate an environment directory.
+            Found a '.flox' directory, but was unable to locate an environment directory.
```

Comment:
nonblocking: we also throw this when `env.json` isn't found, so it would be nice to tweak the wording here and where we define `EnvNotFound`

Reply:
Added a `EnvPointerNotFound` error to be more specific